packages/js/src/indexation.js

```diff
@@ -3,12 +3,12 @@ import jQuery from "jquery";
 
 import Indexation from "./components/Indexation";
 
-const preIndexingActions = {};
-const indexingActions = {};
-
 window.yoast = window.yoast || {};
 window.yoast.indexing = window.yoast.indexing || {};
 
+window.yoast.indexing.preIndexingActions = {};
+window.yoast.indexing.indexingActions = {};
```

Comment:
this order here makes me a bit nervous, though I can see in the "before" that we passed empty objects anyways.

Yet here you override a window object that we just instantiated, which could be problematic.

Reply:
explained in zoom call ✅